client/src/hooks/store/types.ts

```diff
@@ -1,8 +1,13 @@
-import { CombatTarget } from "@/types";
-import { Realm, Structure } from "../helpers/useStructures";
-import { Position } from "@bibliothecadao/eternum";
+export enum Action {
```

Comment:
Name seems a bit too generic.
BattleAction ?

Reply:
Not using it anymore so removed